packages/forma-36-react-components/src/components/CopyButton/CopyButton.css

```diff
@@ -44,7 +44,7 @@
 .CopyButton__TabFocusTrap {
   width: 100%;
   height: 100%;
-  display: flex;
+  display: flex !important;
```

Comment:
Would that work `.CopyButton .CopyButton__button`?

Reply:
Can you clarify your concern?

Reply:
`!important` is a [bad practice](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity#the_!important_exception) and I don't think we should use it to solve css specificity issues, especially if we have control over both components `FocusTrap` and `CopyButton`

Reply:
I fully agree with that but changing the order of class names didn't change anything in the production build, probably a tooling thing, so I'm not sure how else to go about it.

Reply:
<img width="353" alt="Screenshot 2021-04-01 at 11 36 05" src="https://user-images.githubusercontent.com/6163988/113274924-82377680-92de-11eb-9bdf-1b7dc3f3fbfa.png">

Tested this, and it fixes the issue on f36 website. I'm not sure what broke it though 💭

Reply:
Thanks Moe ❤️